### DISCUSSION TITLE
Having problem loading my Tailwindcss project on the browser

### DISCUSSION THREAD

**Discussion by WendahNyiawung** [tailwindlabs/tailwindcss#1887] [Help]
Please i am new to tailwind css  I have created a project with tailwind following instructions from the tailwind site when i click on the index.html file without using an IDE it loads a page without any style on it that is it shows just the default html  [here](https://elegant-ptolemy-a9a01e.netlify.app/) is how it looks like[https://elegant-ptolemy-a9a01e.netlify.app/]( https://elegant-ptolemy-a9a01e.netlify.app/) here is a link to the project on github [https://github.com/WendahNyiawung/bookstore](https://github.com/WendahNyiawung/bookstore) 
thank you

**Comment by estevanmaito** [comment]
Your `dist/index.html` is pointing to resources in the wrong place. Every url/resource is looking for `/dist`, but you are already inside it and there is not a `/dist` folder inside `dist`.

If you open the browser console, you'll see that not only Tailwind styles, but every resource is missed with a 404 error.

**Reply by WendahNyiawung** [reply]
please what do i do

**Reply by estevanmaito** [reply]
Remove every `/dist` inside `/dist/index.html`

So things like this:

```html
<link rel="stylesheet" href="/dist/style.css">
```

Become this:

```html
<link rel="stylesheet" href="/style.css">
```

**Reply by WendahNyiawung** [reply]
I have put all my files in the same directory and i still face the same problem

**Reply by estevanmaito** [reply]
It's not about the place they are, it's about the code inside your `index.html` file, that is located inside the `dist` folder, pointing to the wrong places.

I'm looking at the code you shared in the repo. If you just remove every `/dist` inside that html file, everything would work.

If you could update the code there to reflect your current stage, it would help too.
